package.json

```diff
@@ -1,17 +1,26 @@
 {
-	"license": "MIT",
+	"author": "Hugging Face",
+	"repository": "github:huggingface/huggingface.js",
+	"scripts": {
+		"docs:update": "pnpm --filter 'doc-internal' start",
+		"changeset:release": "pnpm --recursive build && pnpm docs:update && changeset publish",
+		"changeset:version": "changeset version",
+		"format": "prettier --write .eslintrc.cjs && eslint --quiet --fix --ext .cjs,.ts .eslintrc.cjs"
```

Comment:
The format command has changed since, it's `prettier --write . && eslint --quiet --fix --ext .cjs,.ts .`

Reply:
I like it, awesome.

Reply:
It's still wrong after merging